src/cqparts/component.py

```diff
@@ -84,6 +86,15 @@ def exporter(self, exporter_name=None):
         from .codec import get_exporter
         return get_exporter(self, exporter_name)
 
+    def export(self, filepath):
```

Comment:
I didn't implement this originally because I didn't think it would work in all cases.
But I was probably over-thinking it... this does lead to some very clean code for simple export cases.
:+1:

Reply:
I'll just throw a `kwargs` in there for good measure

Reply:
and some docs... docs are my cocain